core/src/plan/expr/function.rs

```diff
@@ -140,6 +140,7 @@ impl Function {
             }
             | Self::Append { expr, value: expr2 }
             | Self::Prepend { expr, value: expr2 }
+            | Self::Take { expr, size: expr2 }
```

Comment:
ref. https://github.com/gluesql/gluesql/pull/1282/commits/6b4aa8a7cd2d6196110ab0317610263ef66955db

could you add an unit test for this `Self::Take`?